apps/utils/locust_tests/common/db.py

```diff
@@ -154,8 +154,8 @@ def get_hashed_mbis(uri: str, table_sample_pct: Optional[float] = None) -> List:
     # by only taking MBI hashes that are distinct in both beneficiaries _and_
     # benficiaries_history
     mbi_query = (
-        f"SELECT beneficiaries.mbi_hash FROM beneficiaries {table_sample_text} "
-        "    INNER JOIN beneficiaries_history "
+        f"SELECT beneficiaries.mbi_hash FROM ccw.beneficiaries {table_sample_text} "
+        "    INNER JOIN ccw.beneficiaries_history "
         "        ON beneficiaries.mbi_hash = beneficiaries_history.mbi_hash "
         "    GROUP BY beneficiaries.mbi_hash "
         "    HAVING count(beneficiaries_history.mbi_hash) = 1 "
```

Comment:
I *think* some of these relations need to be more fully qualified with the `ccw` prefix. If I'm honest, though, I can't seem to get this to execute correctly at all. Still poking at it...

Reply:
> I _think_ some of these relations need to be more fully qualified with the `ccw` prefix. If I'm honest, though, I can't seem to get this to execute correctly at all. Still poking at it...

I'm glad I hedged. Seems like additional qualification here would just be redundant. Should be fine–sorry for the noise.